website/docs/getting-started.mdx

```diff
@@ -121,7 +127,7 @@ cd fleet-infra
 
 ```
 flux create source helm ww-gitops \
-  --url=https://helm.gitops.weave.works \
+  --url=oci://ghcr.io/weaveworks/charts \
```

Comment:
```suggestion
  --url=oci://ghcr.io/weaveworks/charts \
  --interval=60m0s \
```
Same interval for the HelmRelease please

Reply:
should we change intervals in a different PR against live docs? wdyt @ozamosi ?

Reply:
I created a follow-up PR and went through all intervals I could find: https://github.com/weaveworks/weave-gitops/pull/2513